### DISCUSSION TITLE
Notifying with email using MS365 MFA

### DISCUSSION THREAD

**Discussion by MPP-Innovecom** [datalust/seq-tickets#1896] [Help]
Hi There,

With the latest changes made in MS365/AzureAD with MFA being made default for all users and no exclusions,

Does your "notifying with email" documentation (https://docs.datalust.co/docs/formatting-html-email#:~:text=Have%20Seq%20send%20email%20notifications,EmailPlus) configuration work with MFA configured for account sending 'From' address?

As MFA might need to be confirmed everytime the account is logged in.

Do you have any further documentation which include MFA configured accounts used as the from address?

Thanks in advance

Stephen

**Comment by nblumhardt** [comment]
Hi Stephen; unfortunately, no - Microsoft365 supports XOAUTH2 authentication for SMTP, but there isn't yet a Seq app that can perform the necessary token acquisition step prior to authenticating with the SMTP server.

I'm open to giving this a shot, though - what state are things in for you? Are you in a position to help test out a new implementation of the email app adding this? If so, sending as much contextual info as possible to us via support@datalust.co would be the best way to get the ball rolling.

**Reply by nblumhardt** [reply]
Hi again! https://github.com/datalust/seq-app-mail now includes a _Seq.App.Mail.Microsoft365_ app that will be able to send email through Microsoft 365 without requiring any special configuration. It's still under development but should be suitable to try out now.

Hope this helps,
Nick
